### DISCUSSION TITLE
etcd fresh install on Alma Linux 9

### DISCUSSION THREAD

**Discussion by sybrandaltron** [etcd-io/etcd#18275] [Q&A] [answered]
I am setting up a LAB to test PostgreSQL HA Environment on Alma Linux nodes. But I have a problem getting etcd to run install did go as plan but its not starting 
![image](https://github.com/etcd-io/etcd/assets/172015933/73b2fb7b-25d7-4e38-9205-2eef5026e9cf)

![image](https://github.com/etcd-io/etcd/assets/172015933/95a93821-0d1e-4ce4-a75a-9516835a7fb0)
 any help would be appreciated?

**Comment by jmhbnz** [comment]
Hi @sybrandaltron - Thanks for your question, can you please confirm:
- Which version of etcd you are using
- How you are installing it
- What your configuration is
- The startup logs from the etcd process

**Reply by sybrandaltron** [reply, marked as answer]
Hi jmhbnz,

Thanks for your reply, I was able to get etcd running as the problem was SELinux being enabled and in enforcing mode on my system.

Below is the steps and the URL of the instructions I use to do the install

https://medium.com/@buildbot.tech/how-to-install-and-configure-etcd-on-linux-18f1ec5031b

# Define the etcd version
ETCD_VER=v3.5.14

# Choose either URL
GOOGLE_URL=https://storage.googleapis.com/etcd
GITHUB_URL=https://github.com/etcd-io/etcd/releases/download

# Download the etcd
curl -L ${GITHUB_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz

# Extract the package
tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp

# Optionally move the etcd and etcdctl binaries to /usr/local/bin for system-wide use
sudo mv /tmp/etcd-${ETCD_VER}-linux-amd64/etcd* /usr/local/bin/

sudo mv etcd /usr/local/bin

sudo mv etcdctl /usr/local/bin

sudo mkdir /etc/etcd # This is for configuration
 
sudo mkdir /var/lib/etcd # This is for data

sudo groupadd --system etcd

sudo useradd -s /sbin/nologin --system -g etcd etcd
 
sudo chown -R etcd:etcd /var/lib/etcd/

sudo systemctl daemon-reload

sudo systemctl start etcd.service     <-- Got Stuck here
